packages/pds/src/app-view/services/feed/index.ts

```diff
@@ -220,14 +224,19 @@ export class FeedService {
         records.map((p) => p.did),
         requester,
       ),
+      this.embedsForPosts(
+        records.map((p) => p.uri),
+        requester,
+        _depth + 1,
```

Comment:
does this go arbitrarily deep?

so like in the case of a quote post of a quote post of a quote post of an image, this'll return that image?

Reply:
oh wait just saw the _depth > 1 check. so only checks one deep?

Reply:
Exactly, there's a depth check in two places:
 - At the top of the function to ensure it doesn't recurse.
 - When mapping to `embeds` in the view.  The purpose there is to distinguish `embeds` being missing ("embeds are not included" e.g. due to depth) versus empty `[]` ("embeds are included but there are none").